src/lib/components/forms/radio-buttons.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+  export let options: string[];
+  let selectedOption: string;
+</script>
+
+<div>
+  {#each options as option}
+    <input
+      id="radio-{option}"
+      class="radio-button hidden text-center"
+      type="radio"
+      bind:group={selectedOption}
+      value={option}
+    />
+    <label
+      class="m-1 justify-self-start rounded-lg bg-taupe-200 px-5 py-1 text-rose-950"
+      for="radio-{option}"
+    >
```

Comment:
```suggestion
      class="m-1 justify-self-start rounded-lg bg-taupe-200 px-5 py-1 text-rose-950"
      for="radio-{option}"
      class:selected={selectedOption === option}
    >
```

Maybe this would be a bit more readable! What do you think? Then we could remove the `radio-button` class and just do `  label.selected {...}` for the css